tests/test_enterprise/test_signals.py

```diff
@@ -246,11 +248,213 @@ def test_default_content_filter(self, default_content_filter, expected_content_f
             assert enterprise_catalog.content_filter == expected_content_filter
 
 
+@mark.django_db
+class TestPendingEnterpriseAdminUserSignals(unittest.TestCase):
+    """
+    Test signals associated with PendingEnterpriseCustomerAdminUser.
+    """
+
+    def setUp(self):
+        """
+        Setup for `TestPendingEnterpriseAdminUserSignals` test.
+        """
+        self.admin_user = UserFactory(id=2, email='user@example.com')
+        self.enterprise_customer = EnterpriseCustomerFactory()
+        super(TestPendingEnterpriseAdminUserSignals, self).setUp()
```

Comment:
This can probably all go in the class method `setUpTestData` to only be run once

Reply:
Using `setUpTestData` causes tests to fail due to `self.admin_user` not existing in the test case, so I'm opting to keep it as `setUp` for now.

Reply:
👍